lms/static/sass/features/_learner-profile.scss

```diff
@@ -1,10 +1,94 @@
 // lms - application - learner profile
 // ====================
 
-// Table of Contents
-// * +Container - Learner Profile
-// * +Main - Header
-// * +Settings Section
+.learner-achievements {
+    .learner-message {
+        @extend %no-content;
+        margin: $baseline*0.75 0;
+
+        .message-header, .message-actions {
+            text-align: center;
+        }
+
+        .message-actions {
+            margin-top: $baseline/2;
+
+            .btn-brand {
+                color: $white;
+            }
+        }
+    }
+}
+
+.certificate-card {
+    display: flex;
+    flex-direction: row;
+    margin-bottom: $baseline;
+    padding: $baseline/2;
+    border: 1px;
+    border-style: solid;
+    background-color: $white;
+    cursor: pointer;
+
+    &:hover {
+        box-shadow: inset 0px 0px 1px 1px $gray-l2;
+    }
+
+    .card-logo {
+        @include margin-right($baseline);
+        width: 100px;
+        height: 100px;
+
+        @media (max-width: $learner-profile-container-flex) { // Switch to map-get($grid-breakpoints,md) for bootstrap
+            display: none;
+        }
+    }
+
+    .card-content {
+        color: $base-font-color;
+        margin-top: $baseline/2;
+    }
+
+    .card-supertitle {
+        @extend %t-title6;
+        color: $lightest-base-font-color;
+    }
+
+    .card-title {
+        @extend %t-title5;
+        @extend %t-strong;
+        margin-bottom: $baseline/2;
+    }
+
+    .card-text {
+        @extend %t-title8;
+        color: $lightest-base-font-color;
+    }
+
+    &.mode-audit {
+        border-color: $audit-mode-color;
+
+        .card-logo {
+            background-image: url('#{$static-path}/images/certificates/audit.png');
```

Comment:
There is no `audit.png`.  Also, see related comments below about `mode-audit` and card-logo.

Reply:
Good catch. I have an audit.png, but I didn't add it.